blog/_posts/2021-08-09-systematics-normalization-blog.md

```diff
@@ -0,0 +1,58 @@
+
+The aim of my work this summer has been to provide a continuation of the systematics normalization project I began last summer. The goal of this project has been to provide meaningful metrics for phylogenies of different sizes. Much of my work this summer to meet this goal involved testing previous findings against existing models, fleshing out previous experimentation, and formally recording these findings in a research paper.
+
+### **The Short and Sweet**
+
+The quick overview of my project can be summed up in the following. There is a random tree model which just creates randomly generated trees with whatever specifications for size and age you want. It creates these trees thousands of times and each time a tree is created, metrics about it are recorded and stored. After thousands of runs for trees of different sizes and ages, we look at that data and ask, what are the min and max values for this data and where does everything in between lie? We call this a percentile range. After that, we want to test this percentile range, so we run the model again with different random seeds and some other settings and make sure that they always average in the 50th percentile. Then, we make new models and test where those models fall in our percentile range and how those results fit into our other findings. Eventually, this allows us to confidently return to you both a value for your tree metric but also a percentile for that value. Aka, a useful metric for tree comparison. 
+
+### **Background and Definitions:**
+
+#### **Phylogenetic Trees**
+
+A phylogenetic tree is a commonly used abstraction or way of thinking about evolutionary relationships in biology. Phylogenetic trees are used to show evolutionary relationships between organisms. The tree begins with a single group usually and branches when a portion of the population mutates. 
+
+_This is a diagram of a basic phylogenetic tree. In this diagram, the population mutated 4 times (as shown by each divergence of the tree at each node). The final 5 taxa at the end of the tree each have unique genotypes and are considered distinct from one another._ 
+
+![Phylo Tree Diagram]({{ site.baseurl }}/assets/abbywlsn/PhyloTreeDiagram.jpg)
+
+#### **Phylogenetic Diversity**
+
+Phylogenetic Diversity was used as the metric for tree comparison in this project for all of our models. Phylogenetic Diversity is a measure of biodiversity in a population or set of species. We have defined Phylogenetic Diversity as the number of ancestor taxa in a tree plus the number of active taxa minus one. This can be conceptualized as the number of edges in the minimum spanning tree, which happens to be the number of nodes minus one. This metric assumes that all branches from parent to child have a length of one. 
```

Comment:
Feels like a bit of a jump going straight into minimum spanning tree. Could you explain how a minimum spanning tree is a reasonable representation of a phylogentic tree?